<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>glaxnimate::gui::font::FontStyleDialog</class>
 <widget class="QDialog" name="glaxnimate::gui::font::FontStyleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Font Style</string>
  </property>
  <property name="windowIcon">
   <iconset theme="dialog-text-and-font">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="10,4,0">
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_system">
      <attribute name="title">
       <string>System</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="glaxnimate::gui::font::FontStyleWidget" name="widget_system" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_google">
      <attribute name="title">
       <string>Google Fonts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="glaxnimate::gui::font::GoogleFontsWidget" name="google_fonts_widget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_custom">
      <attribute name="title">
       <string>Custom</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="glaxnimate::gui::font::ExternalFontWidget" name="custom_font_widget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="glaxnimate::gui::font::FontPreviewWidget" name="preview" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glaxnimate::gui::font::FontPreviewWidget</class>
   <extends>QWidget</extends>
   <header>widgets/font/font_preview_widget.hpp</header>
   <container>1</container>
   <slots>
    <slot>set_font(QFont)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>glaxnimate::gui::font::FontStyleWidget</class>
   <extends>QWidget</extends>
   <header>widgets/font/font_style_widget.hpp</header>
   <container>1</container>
   <slots>
    <signal>font_changed(QFont)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>glaxnimate::gui::font::GoogleFontsWidget</class>
   <extends>QWidget</extends>
   <header>widgets/font/google_fonts_widget.hpp</header>
   <container>1</container>
   <slots>
    <signal>font_changed(QFont)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>glaxnimate::gui::font::ExternalFontWidget</class>
   <extends>QWidget</extends>
   <header>widgets/font/external_font_widget.hpp</header>
   <container>1</container>
   <slots>
    <signal>font_changed(QFont)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>glaxnimate::gui::font::FontStyleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>746</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>833</x>
     <y>658</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>glaxnimate::gui::font::FontStyleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>696</x>
     <y>658</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget_system</sender>
   <signal>font_changed(QFont)</signal>
   <receiver>preview</receiver>
   <slot>set_font(QFont)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>google_fonts_widget</sender>
   <signal>font_changed(QFont)</signal>
   <receiver>preview</receiver>
   <slot>set_font(QFont)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_font_widget</sender>
   <signal>font_changed(QFont)</signal>
   <receiver>preview</receiver>
   <slot>set_font(QFont)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
